android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -70,10 +73,14 @@
                 else
                     Snackbar.make(v, context.getResources().getString(R.string.no_comments), Snackbar.LENGTH_SHORT).show();
             });
+
+            feedImageView.setOnClickListener(v -> {
+                zoomImage(Utils.parseImageUri(feedItems.get(getPosition()).getFullPicture()));
+            });
         }
     }
 
-    public FeedAdapter(Context context, AdapterCallback adapterCallback, List<FeedItem> feedItems) {
+    public FeedAdapter(Context context, AdapterCallback adapterCallback, ZoomableImageModule zoomableImageModule, List<FeedItem> feedItems) {
```

Comment:
Instead of adding it to constructor, make it optional and ask it as a setter

Reply:
`ZoomableImageModule` is ambiguous. Use name like `OnImageZoomListener` and the method being `onZoom(String)`